base/scopedvalues.jl

```diff
@@ -85,6 +85,10 @@ struct Scope
     values::ScopeStorage
 end
 
+function Scope(parent::Scope)
+    return Scope(parent.values)
+end
```

Comment:
Or should this just forward?
```suggestion
Scope(parent::Scope) = parent
```

Reply:
That's what https://github.com/JuliaLang/julia/pull/53462 does so I'll do that it here.